core/src/locale/he.ts

```diff
@@ -0,0 +1,100 @@
+import type { Localization } from './types'
+
+const locale: Localization = {
+  '*': {
+    prefix: 'בכל',
+    suffix: '',
+    text: 'לא ידוע',
+    '*': {
+      empty: { text: 'בכל {{field.id}}' },
```

Comment:
`{{field.id}}` should only be used in the english translation.
![image](https://github.com/abichinger/vue-js-cron/assets/8122139/569b2361-0598-4287-8c96-ed1c63d77397)

For example, to translate `every month` you can set:
```ts
{
  '*': {
    month: {
      empty: { text: 'your translation' }
    }
  }
}
```